homeassistant/components/renault/sensor.py

```diff
@@ -0,0 +1,104 @@
+"""Support for Renault sensors."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import LENGTH_KILOMETERS, VOLUME_LITERS
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .renault_entities import RenaultCockpitDataEntity, RenaultDataEntity
+from .renault_hub import RenaultHub
+from .renault_vehicle import RenaultVehicleProxy
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities,
+) -> None:
+    """Set up the Renault entities from config entry."""
+    proxy: RenaultHub = hass.data[DOMAIN][config_entry.unique_id]
+    entities = await get_entities(proxy)
+    async_add_entities(entities)
+
+
+async def get_entities(proxy: RenaultHub) -> list[RenaultDataEntity]:
+    """Create Renault entities for all vehicles."""
+    entities = []
+    for vehicle in proxy.vehicles.values():
+        entities.extend(await get_vehicle_entities(vehicle))
+    return entities
+
+
+async def get_vehicle_entities(vehicle: RenaultVehicleProxy) -> list[RenaultDataEntity]:
+    """Create Renault entities for single vehicle."""
+    entities = []
```

Comment:
```suggestion
    entities: list[RenaultDataEntity] = []
```

Reply:
Yep - I saw the issue with mypy. Hopefully I can get that sorted before the weekend.

Reply:
I managed to sort it out last night - including `.strict-typing`

Reply:
Thanks!